eng/common/scripts/modules/ChangeLog-Operations.psm1

```diff
@@ -0,0 +1,109 @@
+# Common Changelog Operations
+
+$RELEASE_TITLE_REGEX = "(?<releaseNoteTitle>^\#+.*(?<version>\b\d+\.\d+\.\d+([^0-9\s][^\s:]+)?)\s(?<releaseStatus>\(Unreleased\)|\(\d{4}-\d{2}-\d{2}\))?)"
+
+# given a CHANGELOG.md file, extract the relevant info we need to decorate a release
+function Extract-ReleaseNotes {
+    param (
+        [Parameter(Mandatory = $true)]
+        [String]$ChangeLogLocation,
+        [String]$VersionString
+    )
+    
+    $ErrorActionPreference = 'Stop'
```

Comment:
Why do we need t set this?

Reply:
I added this to get the call from the msbuild target properly error.

Reply:
It probably is not necessary now that it is a function rather than a stand alone script. I'll look into that.

Reply:
I would remove this from here as it may cause issues with other scripts accidently. If we need this set it should be set at a higher level not in this helper function.